### PR TITLE
Add debian package

### PR DESCRIPTION
**Pull request by Adimote**
Make it a debian package so we can ship it straight to the robots instead of as an interface in the USB sticks

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+
+export PYBUILD_NAME=robot-api
+export PYBUILD_RULES=3.4
+
+%:
+	dh $@ --with python3 --buildsystem=pybuild
+
+override_dh_install:
+	find debian/ -name __pycache__ -type d | xargs -r rm -r
```
Comment (PeterJCLaw) [inline debian/rules]: You might enjoy the `-delete` option to `find`

File: README.md [hunk 2]
```diff
@@ -2,6 +2,35 @@
...
+debuild -uc -us
+
+```
+
+## Instaling the debian package
+
+After building, the .deb file should appear in the parent directory, just run
+
+``` bash
+sudo dpkg -i
```
Comment (PeterJCLaw) [inline README.md]: Does this not need the name of the package file to be passed on the command line?

File: debian/rules (hunk 1 above)
Comment (PeterJCLaw) [reply debian/rules]: (Is there not a better way of excluding these files from being included?)

File: README.md [hunk 3]
```diff
@@ -2,6 +2,38 @@
...
+
+# Install dependencies
+sudo mk-build-deps -ir
+
+# Build the package:
+debuild -uc -us
+
+```
+
+## Instaling the debian package
```
Comment (prophile) [inline README.md]: "Installing"

Comment (Adimote) [reply README.md]: Huh, the commit message was also wrong, it appears I don't know how to spell intsall.

**Pull request merged**